README.md

```diff
@@ -36,7 +36,8 @@ Certain properties of a resource are _semantic_ and have special meaning when us
 * **`createOnlyProperties`**: A property in the `createOnlyProperties` cannot be specified in an **UPDATE** request, and can only be specified in a **CREATE** request. Another way to think about this - these are properties which are 'write-once', such as the [`Engine`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-rds-database-instance.html#cfn-rds-dbinstance-engine) property for an [`AWS::RDS::DBInstance`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-rds-database-instance.html) and if you wish to change such a property on a live resource, you should replace that resource by creating a new instance of the resource and terminating the old one. This is the behaviour CloudFormation follows for all properties documented as _'Update Requires: Replacement'_. An attempt to supply these properties to an **UPDATE** request will produce a runtime error from the handler.
 * **`deprecatedProperties`**: A property in the `deprecatedProperties` is not guaranteed to be present in the response from a **READ** request. These fields will still be accepted as input to **CREATE** and **UPDATE** requests however they may be ignored, or converted to new API forms when outbound service calls are made.
 * **`replacementStrategy`**: As mentioned above, changing a `createOnlyProperty` requires replacement of the resource by creating a new one and deleting the old one. The default CloudFormation replacement behavior is to create a new resource first, then delete the old resource, so as to avoid any downtime. However, some resources are singleton resources, meaning that only one can exist at a time. In this case, it is not possible to create a second resource first, so CloudFormation must Delete first and then Create. Specify either `create_then_delete` or `delete_then_create`. Default value is `create_then_delete`
- * **`taggable`**: A boolean type property which defaults to true, indicating this resource type supports updatable [`tagging property`](https://docs.aws.amazon.com/general/latest/gr/aws_tagging.html). Otherwise, it indicates this resource type does not contain any updatable [`tagging properties`](https://docs.aws.amazon.com/general/latest/gr/aws_tagging.html).
+* **`taggable`**: A boolean type property which defaults to true, indicating this resource type supports updatable [`tagging property`](https://docs.aws.amazon.com/general/latest/gr/aws_tagging.html). Otherwise, it indicates this resource type does not contain any updatable [`tagging properties`](https://docs.aws.amazon.com/general/latest/gr/aws_tagging.html).
+* **`propertyTransform`**: Is a map (Map<String, String>) with the key set to property name and value to set to jsonata transformation function (https://jsonata.org/). This property is used to avoid falsely drifted resources. If the handler transforms the input to the resource to an expected value a transform function can be defined for this property to avoid drift.
```

Comment:
This library doesn't look heavily used outside of JavaScript, so I'm curious how this is implemented

Reply:
Pat, I agree that we need more documentation. Right now the downstream service fix is still in progress and is not ready for customers to use. I am a little apprehensive about adding a whole lot of information without the deployment being complete. Please let me know if you disagree.

Reply:
Is there a good way to indicate an "in progress" property? From a customer perspective: having the documentation available before the implementation, seems like a good way to work backwards and provide feedback on how things might work. I completely understand the other side of that too (and I don't have a strong opinion either way).  

Currently, the only feedback that I have is very minor (although I'm sure more eyes would lead to more feedback), and something you'd probably address when creating more documentation:
- "If the handler transforms the input to the resource to an expected value a transform function can be defined for this property to avoid drift" was a bit hard to parse for me. If I read that correctly it could be rewritten as "If the read handler implementation is causing equivalent values to be incorrectly seen as drifted,  a `propertyTransform` can be applied to the (original) template properties, to make sure that the values being compared are using the same representation. This happens before the drift comparison is done"